spec/classes/redis_spec.rb

```diff
@@ -10,7 +10,11 @@
     when 'FreeBSD'
       '/usr/local/etc/redis.conf'
     when 'RedHat'
-      '/etc/redis.conf'
+      if facts[:operatingsystemmajrelease].to_i > 8
```

Comment:
please don't rely on the legacy topscope facts:
```suggestion
      if facts[:os]['release']['major'].to_i > 8
```

Reply:
I intentionally used legacy ones because these were widely used in unit tests. But I add a commit to replace all of them, instead of fixing only these items. Let me know if you have a different thought.